deps/wazuh_testing/wazuh_testing/tools/s3_package.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+systems = {
+    'rpm': 'rpm',
+    'deb': 'deb',
+    'windows': 'windows',
+    'macos': 'macos',
+    'solaris10': 'solaris10',
+    'solaris11': 'solaris11',
+    'rpm5': 'rpm5',
+    'wpk-linux': 'wpk-linux',
+    'wpk-windows': 'wpk-windows'
+}
+
+architectures = {
+    'i386': 'i386',
+    'sparc': 'sparc',
+    'x86_64': 'x86_64',
+    'amd64': 'amd64',
+    'arm64v8': 'arm64v8',
+    'arm32v7': 'arm32v7'
+}
+
+def get_version(version):
+    """Format the version
+
+    Args:
+        version (string) : String containing the version
+
+    Returns:
+        str: String with the version in a usable format for the tool
+    """
+    tokens = version.split('.')
+    def_version = f"{tokens[0]}.{tokens[1]}"
+    return def_version
+
+
+
+def is_repository(repo):
+    """Check if the directory is a repo type in the S3 server
+
+    Args:
+        repo (string): String with the directory name  
+
+    Returns:
+        boolean: True if the directory is a repo, false otherwise.
+    """
+    repository_list = ['pre-release', 'futures', 'debug', 'trash', 'staging', 'live']
+    non_repository_list = ['warehouse-branches', 'warehouse-pullrequests', 'warehouse-test']
+    is_repo = False
+
+    if repo in repository_list:
+        is_repo = True
+    elif repo in non_repository_list:
+        is_repo = False
+
+    return is_repo
```

Comment:
```suggestion
repository_list = ['pre-release', 'futures', 'debug', 'trash', 'staging', 'live']
non_repository_list = ['warehouse-branches', 'warehouse-pullrequests', 'warehouse-test']
    
if folder_path in repository_list:
    return True
elif folder_path in non_repository_list:
    return False
else:
    raise ValueError(f"{folder_path} is unknown. It was not found in {repository_list} or {non_repository_list}")
```

Reply:
This requested change can be reviewed in the commit : [5c3a34d2757275d66bb9951667083965a0452fab](https://github.com/wazuh/wazuh-qa/commit/5c3a34d2757275d66bb9951667083965a0452fab)